server/models/session.js

```diff
@@ -111,7 +109,11 @@ Session.schema = Joi.object().keys({
     _id: Joi.object(),
     userId: Joi.string().required(),
     key: Joi.string().required(),
-    time: Joi.date().required()
+    time: Joi.date().required(),
+    lastLogin: Joi.date().required(),
+    ip: Joi.string().required(),
+    browser: Joi.string().required(),
+    os: Joi.string().required()
```

Comment:
How should we handle someone not passing in a `user-agent` header? Let's say they logged in via Postman or another custom client?

Reply:
Logged though postman native app, with recent change, returns back:
```json
"browser": "PostmanRuntime/6.4.1",
"os": "Other 0.0.0",
```

Reply:
Ok maybe Postman was a bad example 😆, I'm curious though what happens if a `user-agent` header isn't provided.

Reply:
I think it just says "Other" from the parser as Browser and os is blank